knowledge-base/grid-edit-selected-item.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Edit the Selected Item
+description: How to edit the Selected Item.
```

Comment:
consider adding somethign like "from the toolbar" or otherwise include command buttons in the toolbar in the title and/or description